src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,15 @@
+package duke.exception;
+
+/**
+ * This class represents exceptions created specific to the Duke bot.
+ */
+public class DukeException extends Exception {
+    /**
+     * Constructor to create a Duke exception.
+     *
+     * @param msg The error message of the exception.
+     */
+    public DukeException(String msg) {
+        super(msg);
```

Comment:
Correct me if I am wrong. If the subclasses are just calling super, perhaps we do not really need this?